app.js

```diff
@@ -18,8 +19,10 @@ if (answer1 === 'yes' || answer1 === 'y') {
     alert('Wrooong! I was born in November.');
 } else {
     alert('Please answer with "yes/y" or "no/n".');
-};
-
+}
+}
+question1();
```

Comment:
It's a good idea to organize your code better, when other people are working on your code they may be looking for particular things, and that gets harder when your function calls are between the function definitions. A good standard is to put your global variables first, followed by your function definitions, and lastly your function calls.